gaea/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
```

Comment:
Because this definition file needs to be used in multiple NPM packages, including gaea-editor gaea-preview gaea-web-components and gaea-native-components..